pydantic_ai_slim/pydantic_ai/_cli.py

```diff
@@ -0,0 +1,170 @@
+from importlib.metadata import version
+from pathlib import Path
+from typing import Literal, TypedDict, cast
+
+try:
+    from prompt_toolkit import PromptSession
+    from prompt_toolkit.auto_suggest import AutoSuggestFromHistory
+    from prompt_toolkit.history import FileHistory
+    from rich.console import Console, ConsoleOptions, RenderResult
+    from rich.live import Live
+    from rich.markdown import CodeBlock, Markdown
+    from rich.syntax import Syntax
+    from rich.text import Text
+except ImportError as _import_error:
+    raise ImportError(
+        'Please install `rich` and `prompt-toolkit` to use the PydanticAI CLI, '
+        "you can use the `cli` optional group — `pip install 'pydantic-ai-slim[cli]'`"
+    ) from _import_error
+
+from pydantic_ai.agent import Agent
+from pydantic_ai.messages import ModelMessage, PartDeltaEvent, TextPartDelta
+
+__version__ = version('pydantic-ai')
+
+
+class SimpleCodeBlock(CodeBlock):
+    def __rich_console__(self, console: Console, options: ConsoleOptions) -> RenderResult:
+        code = str(self.text).rstrip()
+        yield Text(self.lexer_name, style='dim')
+        yield Syntax(code, self.lexer_name, theme=self.theme, background_color='default', word_wrap=True)
+        yield Text(f'/{self.lexer_name}', style='dim')
+
+
+Markdown.elements['fence'] = SimpleCodeBlock
+
+
+def cli() -> int:
+    parser = argparse.ArgumentParser(
+        prog='PydAI',
+        description=f"""\
+PydanticAI CLI v{__version__}
+
+Special prompt:
+* `multiline` - toggle multiline mode
+""",
+    )
+    parser.add_argument('prompt', nargs='?', help='AI Prompt, if omitted fall into interactive mode')
+    # Default model, if omitted it will default to openai:gpt-4o.
+    parser.add_argument(
+        'model', nargs='?', help='Model to use, if omitted it will default to openai:gpt-4o', default='openai:gpt-4o'
+    )
+    parser.add_argument('--no-stream', action='store_true', help='Whether to stream responses from OpenAI')
+    parser.add_argument('--version', action='store_true', help='Show version and exit')
+
+    args = parser.parse_args()
+
+    console = Console()
+    console.print(f'PydAI - PydanticAI CLI v{__version__}', style='green bold', highlight=False)
+    if args.version:
+        return 0
+
+    now_utc = datetime.now(timezone.utc)
+    tzname = now_utc.astimezone().tzinfo.tzname(now_utc)  # type: ignore
+    agent = Agent(
+        model=args.model or 'openai:gpt-4o',
+        system_prompt=f"""\
+Help the user by responding to their request, the output should be concise and always written in markdown.
+The current date and time is {datetime.now()} {tzname}.
+The user is running {sys.platform}.""",
+    )
+
+    stream = not args.no_stream
+
+    if prompt := cast(str, args.prompt):
+        try:
+            asyncio.run(ask_agent(agent, prompt, stream, console))
+        except KeyboardInterrupt:
+            pass
+        return 0
+
+    history = Path().home() / '.prompt-history.txt'
```

Comment:
should this be pydai-specific in some way? Feels a bit presumptuous to not put any reference to pydai in a file we are creating in your home directory, and in a file that presumably is not intended to be shared with other things. Or maybe I'm wrong about that and it should be shared with other things?